### PR TITLE
legacyrpc: get rid of extraneous comment in methods and remove dcrrpcclient from packages

### PR DESCRIPTION
**Pull request by githubsands**
As told by jrick this problem has been solved.

### CODE REVIEW

File: rpc/legacyrpc/methods.go [hunk 1]
```diff
@@ -1115,8 +1115,7 @@ func getMultisigOutInfo(s *Server, icmd interface{}) (interface{}, error) {
 // getNewAddress handles a getnewaddress request by returning a new
 // address for an account.  If the account does not exist an appropiate
 // error is returned.
-// TODO: Follow BIP 0044 and warn if number of unused addresses exceeds
-// the gap limit.
+
```
Comment (jrick) [inline rpc/legacyrpc/methods.go]: remove this newline

**Comment by jrick** [comment]
Remove the commit that changes the help. It is incorrect, and unrelated to this change.

**Comment by githubsands** [comment]
didn't mean to put the Update helpdescs_en_US.go in here. methods is all good to go.

**Comment by githubsands** [comment]
Also updated to remove dcrrpcclient from the methods.go packages.  dcrprcclient is unused, contents were migrated to dcrd.

**Comment by jrick** [comment]
Closing -- this doesn't even compile due to the inclusion of a completely unnecessary (and incorrect) change.

**Pull request closed without merging**